### ISSUE TITLE
'Summary' tags (OpenAudible 3.0.1.4 Beta)

### ISSUE THREAD

**Issue by RandmTask** [openaudible/openaudible#671]
Hi,

Thanks for the great program.

My use case is for PLEX and Prologue

I have OpenAudible set to convert to M4B on files from the Audible.com.au store

The **'Summary'** (Description/Synopsis of the book) does not show up in PLEX or Prologue, I need to manually use a program called 'Tag Editor' on a Mac to paste in the field **'Podcast Description'** from OA or copy and paste from Audible so that I have a description.

As an aside, the '**Comment**' field shows just one line of the **'Summary'**
ie '**Comment**' tag shows: 
**_Magical retellings of three classic fairy tales, read by Tamsin Greig and Stephen Mangan...._**

Whereas '**Summary**' in OA shows: 
**_Magical retellings of three classic fairy tales, read by Tamsin Greig and Stephen Mangan._**

**_“Aladdin”_**

**_Aladdin is a tailor’s son who dreams of making his fortune and marrying the Sultan’s daughter, but is tricked into becoming apprentice to a cunning magician. He needs Aladdin to do a very special job - climb down into a deep, dark cave and fetch an oil lamp…._**

Hope that makes sense!

tl;dr Could we please have the Summary copied into the Podcast Description / Description field

**Comment by openaudible** [comment]
I found a Tag Editor Free but it didn't have a summary tag that I could find, and it also didn't load my m4b files. (It liked m4a) I might need you to drag and drop an M4B that you modified to ...

I found this guide here that looks pretty good.. 
https://github.com/seanap/Plex-Audiobook-Guide
And will try to implement some changes and report back.

**Comment by RandmTask** [comment]
Thanks for your reply. Sorry I wasn't clear!

**Summary** is what appears in the OpenAudible app

![Summary](https://user-images.githubusercontent.com/7912017/127315321-9d3abf9b-43ef-4155-9bfa-85e5059f07a6.png)

In Tag Editor (and other editors I've tried) the **Comment** tag is a shortened version of what appears in the Summary section of OpenAudible.

It would be great if we could get the complete text of the **Summary** section to appear in the **Podcast Description** tag as this is what PLEX and Prologue read. At the moment I am copying from OpenAudible manually into the **Podcast Description** tag.

![Screen Shot 2021-07-28 at 3 45 42 pm](https://user-images.githubusercontent.com/7912017/127317293-f5b2dbdd-f243-4dc2-9df0-27b41509ca8d.png)

FYI **Tag Editor** and **Tag Editor Free** are by the same company but **Tag Editor Free** hasn't been updated in 6 years which probably explains why m4b didn't work. But I get the same behaviour in M4A files.

**Comment by RandmTask** [comment]
Also,

Also, there seems to be a bug in that the Summary changes when it is downloaded, compared to how it was when it hasn't been downloaded yet. Ive seen this with multiple books.

It looks like the Summary before the title has been downloaded is the preferential one to keep, as it is longer and properly spaced.

![Screen Shot 2021-07-28 at 3 25 50 pm](https://user-images.githubusercontent.com/7912017/127317701-3b9553e3-4c08-4d2e-8ad7-560325e257a0.png)


![Screen Shot 2021-07-28 at 3 44 42 pm](https://user-images.githubusercontent.com/7912017/127317715-2e7bff97-f924-4069-8ff0-8a3fd50846f4.png)

**Comment by openaudible** [comment]
Good feedback. I’d like to get the tags figured out for Plex. I think the full summary (minus html tags) needs to go in the “desc” tag. 

Is the AudioBook Plex plugin required to view this meta data? Will need help testing or setting up a test Plex. 

Thanks for the report on the summary change. Assume there is an existing summary tag in the audio file and we are updating it when we read it in. Will explore.

**Comment by RandmTask** [comment]
Hi,

I dont believe the Audiobook plex plugin is required to view the metadata. I have looked at the link provided but never managed to get it set up correctly.

https://github.com/seanap/Plex-Audiobook-Guide

It has the following list of tags they use, perhaps some of those could be implemented if it's not outside the scope of your program?

<img width="506" alt="Screen Shot 2021-07-29 at 11 36 56 am" src="https://user-images.githubusercontent.com/7912017/127451560-95f98907-54eb-4fa1-853f-a58753c166c8.png">
<img width="510" alt="Screen Shot 2021-07-29 at 11 37 06 am" src="https://user-images.githubusercontent.com/7912017/127451579-193dcc54-3681-4b31-b7c6-242be2303c20.png">

**Comment by RandmTask** [comment]
> Good feedback. I’d like to get the tags figured out for Plex. I think the full summary (minus html tags) needs to go in the “desc” tag.
> 
> Is the AudioBook Plex plugin required to view this meta data? Will need help testing or setting up a test Plex.
> 

More than happy to test it out :)

**Comment by openaudible** [comment]
I found plex was confused by my m4b's. I played around with some tags and it appears to be working.

https://openaudible.org/beta

I also fixed the "Summary" update you found. 

You will need to delete all m4b files, do a full library rescan (to reset the summary to the long summary) then convert all to m4b. 

Hopefully that will work!

**Comment by RandmTask** [comment]
Fantastic.

Both M4B's and M4A's work well in PLEX, just the line spacing is missing, so it kind of all runs together

![Screen Shot 2021-07-30 at 7 55 46 pm](https://user-images.githubusercontent.com/7912017/127679624-3dc3e69d-e082-4336-903b-ef362131adb4.png)

**Comment by openaudible** [comment]
Hmm. That's the tag editor. It looks like linefeeds (\n) are being added to the tag.. are they displayed correctly in plex? Looking at it.

**Comment by openaudible** [comment]
I posted a new version that keeps the exiting html tags in the comments/summary tags. Feel free to let me know which is better.

**Comment by RandmTask** [comment]
> Hmm. That's the tag editor. It looks like linefeeds (\n) are being added to the tag.. are they displayed correctly in plex? Looking at it. I can try keeping the html tags in the summary..

Nope

This is from PLEX (Beta 5)

![Screen Shot 2021-07-31 at 11 26 51 am](https://user-images.githubusercontent.com/7912017/127732437-92dc7610-10bb-4421-b19d-286e3e725263.png)

**Comment by RandmTask** [comment]
> I posted a new version that keeps the exiting html tags in the comments/summary tags. Feel free to let me know which is better.

Unfortunately doesn't seem like PLEX likes those tags.

This is from Beta 6

![Screen Shot 2021-07-31 at 11 26 35 am](https://user-images.githubusercontent.com/7912017/127732449-729b88c7-d2e8-4b6e-a7c5-5cf14b68a3f7.png)


At the moment all I am doing is just copying and pasting from **Summary** in the OpenAudible App to the **Description** tag in Tag Editor. Can a similar copy/paste happen behind the scenes?

**Comment by openaudible** [comment]
Got the Plex beta and think I fixed it. Try Beta 7.

**Comment by openaudible** [comment]
Posted new 3.0.2 version. Feel free to repopen or report anything else. 

Thanks for your help.

**Comment by RandmTask** [comment]
Thanks just installed 3.0.2.1beta and it seems to be working great now! Thanks for that!
